test/extensions/filters/http/common/fuzz/uber_per_filter.cc

```diff
@@ -44,6 +44,32 @@ void addBookstoreProtoDescriptor(Protobuf::Message* message) {
 }
 } // namespace
 
+void UberFilterFuzzer::guideAnyProtoType(test::fuzz::HttpData* mutable_data, uint choice) {
+  // These types are request/response from the test Bookstore service
+  // for the gRPC Transcoding filter.
+  static const std::vector<std::string> expected_types = {
+      "type.googleapis.com/bookstore.ListShelvesResponse",
```

Comment:
For later, I wonder if these types can be picked up via reflection.

Reply:
Good point, I didn't think of that. I did some initial research and it seems feasible. Lets do that in another PR later, if we end up needing to fuzz test with proto data for any other filters.